tsconfig.json

```diff
@@ -1,18 +1,26 @@
 {
   "compilerOptions": {
-    "target": "esnext",
```

Comment:
any reason why we want to transpile down to es2017?

Reply:
Orignally it was to support the enum outputs in the .js but since moving to const enums this is no longer needed, amending